client/components/mma/accountoverview/CancelledProductCard.tsx

```diff
@@ -100,7 +99,7 @@ export const CancelledProductCard = ({
 						<div css={wideButtonLayoutCss}>
 							{showSubscribeAgainButton && (
 								<LinkButton
-									href="https://support.theguardian.com/uk/subscribe"
+									href={`https://support.theguardian.com/${specificProductType.checkoutUrlPart}`}
```

Comment:
this ends up with a double slash as we have the slash both here and in the checkoutUrlPart
https://support.theguardian.com//support
Then that causes a 404